### ISSUE TITLE
TFLite model exported by yolv8 can run normally, but TFlite exported by yolo11 can't run normally.

### ISSUE THREAD

**Issue by jiokul** [ultralytics/ultralytics#20961]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/orgs/ultralytics/discussions) and found no similar questions.

### Question

TFLite model exported by yolv8 can run normally, but TFlite exported by yolo11 can't run normally.
I run on Android11.
errorlog
01-01 00:42:57.984 14839-14865 TFLite                  com.panasonic.rsepoc.tensorlow       I  ✅ GPU 加速可用
2022-01-01 00:42:57.985 14839-14865 inference_plate         com.panasonic.rsepoc.tensorlow       D  initialize: GPU
2022-01-01 00:42:57.987 14839-14865 tflite                  com.panasonic.rsepoc.tensorlow       I  Created TensorFlow Lite delegate for GPU.
2022-01-01 00:42:57.987 14839-14865 inference_plate         com.panasonic.rsepoc.tensorlow       D  initialize: 1
2022-01-01 00:42:58.015 14839-14865 InterpreterApi          com.panasonic.rsepoc.tensorlow       I  Loaded native library: tensorflowlite_jni
2022-01-01 00:42:58.017 14839-14865 InterpreterApi          com.panasonic.rsepoc.tensorlow       I  Didn't load native library: tensorflowlite_jni_gms_client
2022-01-01 00:42:58.018 14839-14865 tflite                  com.panasonic.rsepoc.tensorlow       I  Initialized TensorFlow Lite runtime.
2022-01-01 00:42:58.029 14839-14865 tflite                  com.panasonic.rsepoc.tensorlow       I  Replacing 857 out of 857 node(s) with delegate (TfLiteGpuDelegateV2) node, yielding 1 partitions for the whole graph.
2022-01-01 00:42:59.629 14839-14855 System                  com.panasonic.rsepoc.tensorlow       W  A resource failed to call close. 
2022-01-01 00:42:59.920 14839-14865 libc                    com.panasonic.rsepoc.tensorlow       A  Fatal signal 11 (SIGSEGV), code 1 (SEGV_MAPERR), fault addr 0x18 in tid 14865 (Thread-3), pid 14839 (sepoc.tensorlow)
2022-01-01 00:43:00.222 14876-14876 DEBUG                   crash_dump64                         A  pid: 14839, tid: 14865, name: Thread-3  >>> com.panasonic.rsepoc.tensorlow <<<
2022-01-01 00:43:00.380 14876-14876 DEBUG                   crash_dump64                         A        #17 pc 000000000019401c  /data/app/~~VRBC9BS1ihF33khYuW4RaQ==/com.panasonic.rsepoc.tensorlow-yM9impeP397rgS6wV_plRw==/base.apk!libtensorflowlite_gpu_jni.so (offset 0x5544000)
2022-01-01 00:43:00.380 14876-14876 DEBUG                   crash_dump64                         A        #18 pc 0000000000193e68  /data/app/~~VRBC9BS1ihF33khYuW4RaQ==/com.panasonic.rsepoc.tensorlow-yM9impeP397rgS6wV_plRw==/base.apk!libtensorflowlite_gpu_jni.so (offset 0x5544000)
2022-01-01 00:43:00.380 14876-14876 DEBUG                   crash_dump64                         A        #19 pc 0000000000188bcc  /data/app/~~VRBC9BS1ihF33khYuW4RaQ==/com.panasonic.rsepoc.tensorlow-yM9impeP397rgS6wV_plRw==/base.apk!libtensorflowlite_gpu_jni.so (offset 0x5544000)
2022-01-01 00:43:00.380 14876-14876 DEBUG                   crash_dump64                         A        #20 pc 00000000000b0328  /data/app/~~VRBC9BS1ihF33khYuW4RaQ==/com.panasonic.rsepoc.tensorlow-yM9impeP397rgS6wV_plRw==/base.apk!libtensorflowlite_gpu_jni.so (offset 0x5544000)
2022-01-01 00:43:00.380 14876-14876 DEBUG                   crash_dump64                         A        #21 pc 000000000009bafc  /data/app/~~VRBC9BS1ihF33khYuW4RaQ==/com.panasonic.rsepoc.tensorlow-yM9impeP397rgS6wV_plRw==/base.apk!libtensorflowlite_gpu_jni.so (offset 0x5544000)
2022-01-01 00:43:00.380 14876-14876 DEBUG                   crash_dump64                         A        #22 pc 000000000009b1fc  /data/app/~~VRBC9BS1ihF33khYuW4RaQ==/com.panasonic.rsepoc.tensorlow-yM9impeP397rgS6wV_plRw==/base.apk!libtensorflowlite_gpu_jni.so (offset 0x5544000)
2022-01-01 00:43:00.380 14876-14876 DEBUG                   crash_dump64                         A        #23 pc 000000000008ad9c  /data/app/~~VRBC9BS1ihF33khYuW4RaQ==/com.panasonic.rsepoc.tensorlow-yM9impeP397rgS6wV_plRw==/base.apk!libtensorflowlite_gpu_jni.so (offset 0x5544000)
2022-01-01 00:43:00.381 14876-14876 DEBUG                   crash_dump64                         A        #24 pc 000000000002ddd4  /data/app/~~VRBC9BS1ihF33khYuW4RaQ==/com.panasonic.rsepoc.tensorlow-yM9impeP397rgS6wV_plRw==/base.apk!libtensorflowlite_gpu_jni.so (offset 0x5544000)
2022-01-01 00:43:00.381 14876-14876 DEBUG                   crash_dump64                         A        #25 pc 000000000002c5b0  /data/app/~~VRBC9BS1ihF33khYuW4RaQ==/com.panasonic.rsepoc.tensorlow-yM9impeP397rgS6wV_plRw==/base.apk!libtensorflowlite_gpu_jni.so (offset 0x5544000)
2022-01-01 00:43:00.381 14876-14876 DEBUG                   crash_dump64                         A        #26 pc 00000000002d2b38  /data/app/~~VRBC9BS1ihF33khYuW4RaQ==/com.panasonic.rsepoc.tensorlow-yM9impeP397rgS6wV_plRw==/base.apk!libtensorflowlite_jni.so (offset 0x597c000)
2022-01-01 00:43:00.381 14876-14876 DEBUG                   crash_dump64                         A        #27 pc 00000000002d2658  /data/app/~~VRBC9BS1ihF33khYuW4RaQ==/com.panasonic.rsepoc.tensorlow-yM9impeP397rgS6wV_plRw==/base.apk!libtensorflowlite_jni.so (offset 0x597c000)
2022-01-01 00:43:00.381 14876-14876 DEBUG                   crash_dump64                         A        #28 pc 00000000002d2184  /data/app/~~VRBC9BS1ihF33khYuW4RaQ==/com.panasonic.rsepoc.tensorlow-yM9impeP397rgS6wV_plRw==/base.apk!libtensorflowlite_jni.so (offset 0x597c000)
2022-01-01 00:43:00.381 14876-14876 DEBUG                   crash_dump64                         A        #29 pc 000000000002c3d8  /data/app/~~VRBC9BS1ihF33khYuW4RaQ==/com.panasonic.rsepoc.tensorlow-yM9impeP397rgS6wV_plRw==/base.apk!libtensorflowlite_gpu_jni.so (offset 0x5544000)
2022-01-01 00:43:00.381 14876-14876 DEBUG                   crash_dump64                         A        #30 pc 00000000002d6d6c  /data/app/~~VRBC9BS1ihF33khYuW4RaQ==/com.panasonic.rsepoc.tensorlow-yM9impeP397rgS6wV_plRw==/base.apk!libtensorflowlite_jni.so (offset 0x597c000)
2022-01-01 00:43:00.381 14876-14876 DEBUG                   crash_dump64                         A        #31 pc 00000000002d7300  /data/app/~~VRBC9BS1ihF33khYuW4RaQ==/com.panasonic.rsepoc.tensorlow-yM9impeP397rgS6wV_plRw==/base.apk!libtensorflowlite_jni.so (offset 0x597c000)
2022-01-01 00:43:00.381 14876-14876 DEBUG                   crash_dump64                         A        #32 pc 00000000002c2a84  /data/app/~~VRBC9BS1ihF33khYuW4RaQ==/com.panasonic.rsepoc.tensorlow-yM9impeP397rgS6wV_plRw==/base.apk!libtensorflowlite_jni.so (offset 0x597c000)
2022-01-01 00:43:00.381 14876-14876 DEBUG                   crash_dump64                         A        #33 pc 00000000002c7188  /data/app/~~VRBC9BS1ihF33khYuW4RaQ==/com.panasonic.rsepoc.tensorlow-yM9impeP397rgS6wV_plRw==/base.apk!libtensorflowlite_jni.so (offset 0x597c000)
2022-01-01 00:43:00.381 14876-14876 DEBUG                   crash_dump64                         A        #34 pc 00000000002c7cb8  /data/app/~~VRBC9BS1ihF33khYuW4RaQ==/com.panasonic.rsepoc.tensorlow-yM9impeP397rgS6wV_plRw==/base.apk!libtensorflowlite_jni.so (offset 0x597c000)
2022-01-01 00:43:00.381 14876-14876 DEBUG                   crash_dump64                         A        #35 pc 0000000000024f9c  /data/app/~~VRBC9BS1ihF33khYuW4RaQ==/com.panasonic.rsepoc.tensorlow-yM9impeP397rgS6wV_plRw==/base.apk!libtensorflowlite_jni.so (offset 0x597c000) (Java_org_tensorflow_lite_NativeInterpreterWrapper_createInterpreter+472)
2022-01-01 00:43:00.382 14876-14876 DEBUG                   crash_dump64                         A        #43 pc 00000000004b2204  /data/app/~~VRBC9BS1ihF33khYuW4RaQ==/com.panasonic.rsepoc.tensorlow-yM9impeP397rgS6wV_plRw==/base.apk (offset 0x5d16000) (org.tensorflow.lite.NativeInterpreterWrapper.init+256)
2022-01-01 00:43:00.382 14876-14876 DEBUG                   crash_dump64                         A        #46 pc 00000000004b1e96  /data/app/~~VRBC9BS1ihF33khYuW4RaQ==/com.panasonic.rsepoc.tensorlow-yM9impeP397rgS6wV_plRw==/base.apk (offset 0x5d16000) (org.tensorflow.lite.NativeInterpreterWrapper.<init>+146)
2022-01-01 00:43:00.382 14876-14876 DEBUG                   crash_dump64                         A        #49 pc 00000000004b17d0  /data/app/~~VRBC9BS1ihF33khYuW4RaQ==/com.panasonic.rsepoc.tensorlow-yM9impeP397rgS6wV_plRw==/base.apk (offset 0x5d16000) (org.tensorflow.lite.NativeInterpreterWrapperExperimental.<init>)
2022-01-01 00:43:00.383 14876-14876 DEBUG                   crash_dump64                         A        #52 pc 00000000004b1600  /data/app/~~VRBC9BS1ihF33khYuW4RaQ==/com.panasonic.rsepoc.tensorlow-yM9impeP397rgS6wV_plRw==/base.apk (offset 0x5d16000) (org.tensorflow.lite.Interpreter.<init>+4)
2022-01-01 00:43:00.383 14876-14876 DEBUG                   crash_dump64                         A        #55 pc 0000000000003108  /data/app/~~VRBC9BS1ihF33khYuW4RaQ==/com.panasonic.rsepoc.tensorlow-yM9impeP397rgS6wV_plRw==/base.apk (offset 0x53db000) (com.panasonic.rsepoc.tensorlow.yolo.yoloInference_dms.initialize+220)
2022-01-01 00:43:00.383 14876-14876 DEBUG                   crash_dump64                         A        #58 pc 0000000000001a2c  /data/app/~~VRBC9BS1ihF33khYuW4RaQ==/com.panasonic.rsepoc.tensorlow-yM9impeP397rgS6wV_plRw==/base.apk (offset 0x5976000) (com.panasonic.rsepoc.tensorlow.MainActivity.lambda$onCreate$3$com-panasonic-rsepoc-tensorlow-MainActivity+64)
2022-01-01 00:43:00.383 14876-14876 DEBUG                   crash_dump64                         A        #61 pc 0000000000001350  /data/app/~~VRBC9BS1ihF33khYuW4RaQ==/com.panasonic.rsepoc.tensorlow-yM9impeP397rgS6wV_plRw==/base.apk (offset 0x5976000) (com.panasonic.rsepoc.tensorlow.MainActivity$$ExternalSyntheticLambda3.run+4)
---------------------------- PROCESS ENDED (14839) for package com.panasonic.rsepoc.tensorlow ----------------------------

**Comment by UltralyticsAssistant** [comment]
👋 Hello @jiokul, thank you for bringing this to our attention and for using Ultralytics 🚀! This is an automated response to help get your issue addressed as quickly as possible. An Ultralytics engineer will follow up with you soon.

We recommend checking out our [Docs](https://docs.ultralytics.com/) for detailed guidance on export and deployment, as well as our [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples.

If this is a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) (MRE)—including your export and inference code, TFLite conversion command, and any relevant model files or logs—to help us debug the issue more efficiently.

If this is a custom training or deployment ❓ Question, please share details such as dataset samples, export parameters, and any specific steps you followed, and double-check our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/).

For additional help, join the Ultralytics community:
- Real-time chat: [Discord](https://discord.com/invite/ultralytics) 🎧
- In-depth discussions: [Discourse](https://community.ultralytics.com/)
- Share and learn: [Subreddit](https://reddit.com/r/Ultralytics)

## Upgrade

Please upgrade to the latest `ultralytics` package and ensure all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) are installed in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/):

```bash
pip install -U ultralytics
```

## Environments

YOLO can run in any of these up-to-date, verified environments (with dependencies like [CUDA](https://developer.nvidia.com/cuda-zone)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/), and [PyTorch](https://pytorch.org/) preinstalled):

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM: [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI: [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**: [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are currently passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

Thank you for your patience and for helping us improve Ultralytics!

**Comment by Y-T-G** [comment]
You can try exporting with latest Ultralytics
